18F/identity-idp
# LG-7521 Remove Double Rule

## 🎫 Ticket
[LG-7521](https://cm-jira.usa.gov/browse/LG-7521) Adjust component to avoid appearance of a double rule

## 🛠 Summary of changes
Remove double line below Post Office search results (/verify/document_capture#location on Find a participating Post Office)

## 📜 Testing Plan
1. Create an account
2. Attempt to verify your identity via /verify
3. Upload front/back of ID to fail identity verification
4. Click Try in person and Continue to get to Find a participating Post Office (via /verify/document_capture#location)
5. Enter text in the input field and click Search
6. Scroll to the bottom of the page. You should not see a double line below the search results
7. Poke through app, nothing else should be effected. 

## 👀 Screenshots
<details>
<summary>Before:</summary>
<img width="666" alt="Screenshot 2023-05-17 at 8 36 48 AM" src="https://github.com/18F/identity-idp/assets/125507397/59fd3cba-eeea-4a09-a446-d913424d1c68">
</details>

<details>
<summary>After:</summary>
<img width="661" alt="Screenshot 2023-05-17 at 8 35 19 AM" src="https://github.com/18F/identity-idp/assets/125507397/72f6bbda-3908-4483-9124-0bfc7d4e6732">
</details>
